dotnet/src/SemanticKernel/Configuration/KernelConfig.cs

```diff
@@ -285,30 +270,30 @@ public BackendConfig GetCompletionBackend(string? label)
 
         throw new KernelException(
             KernelException.ErrorCodes.BackendNotFound,
-            $"Completion backend not found: {label}. No default backend available.");
+            $"Completion backend not found with label: {label} and no default completion backend is available.");
     }
 
     /// <summary>
-    /// Get the backend configuration matching the given embeddings backend label.
+    /// Get the embeddings backend configuration matching the given label or the default if a label is not provided or not found.
```

Comment:
"...or not found"
This summary is not technically correct... if you provide a bad label name and it's not found, it will not return the default. It'll only try to get the default if the input is null/empty. (same with the GetCompletionBackend)